nsv13/code/modules/munitions/ammunition/cannon_ammo.dm

```diff
@@ -1,39 +1,29 @@
-/obj/item/ammo_box/magazine/light_cannon
+/obj/item/ammo_box/magazine/pdc/light_cannon
```

Comment:
Why is this a subtype of PDC?

Reply:
so it can inherit the variables and the proc that I added

Reply:
You are aware that means these can be loaded into PDCs, yes?

Reply:
is that how that works
well, I'm gonna need to change that

Reply:
Did some testing, no subtype of ```/magazine/pdc/``` can be loaded into the rebranded 50cal PDCs, other than ```/magazine/pdc/fiftycal```.
In the case of the "PDC Loading Rack", uhh I don't think we even use those anymore? but I'll go ahead and fix it anyway.